src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -2287,6 +2287,69 @@ static void Main(string[] args)
             VerifyOperationTreeAndDiagnosticsForTest<VariableDeclarationSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void UsingStatement_DefaultDisposeArguments()
```

Comment:
>UsingStatement_DefaultDisposeArguments [](start = 20, length = 38)

I am somewhat confused about placement of this test. It looks like it tests IUsingOperation node, but we have a dedicated test file for it.